src/test/java/organice/logic/commands/ListCommandTest.java

```diff
@@ -19,21 +22,64 @@
 
     private Model model;
     private Model expectedModel;
+    private Type type;
```

Comment:
A suggestion is to declare constants for Type("patient"), Type("donor"), Type("doctor") and use them in your tets.

Reply:
> 
> 
> A suggestion is to declare constants for Type("patient"), Type("donor"), Type("doctor") and use them in your tets.

Yes, use stubs ty

Reply:
Done.